App/app.rb

```diff
@@ -0,0 +1,128 @@
+# rubocop:disable all
+
+require './Entities/student'
+require './Entities/teacher'
+require './Relationship/book'
+require './Relationship/rental'
+
+class App
+  def initialize(parent)
+    @parent = parent
+    @books_list = []
+    @people_list = []
+    @rentals_list = []
+  end
+
+  def list_all_books
+    if @books_list.empty?
+      puts 'No record found! Add some books...'
+    else
+      puts 'Available books in the library'
+      @books_list.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+    end
+    @parent.show_menu
+  end
+
+  def list_all_people
+    if @people_list.empty?
+      puts 'No record found! Add a person...'
+    else
+      puts 'All people in the library'
+      @people_list.each do |person|
+        puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      end
+    end
+    @parent.show_menu
+  end
```

Comment:
- Please note that I encounter some errors when I try to list all people. I'd suggest you check the error message for more insight into what went wrong 👍🏽.
![Screenshot from 2023-01-14 22-01-17](https://user-images.githubusercontent.com/44624138/212496789-0883f3ee-e8de-4ddd-87fc-211ac5c47a43.png)